bindings/java/src/main/java/tech/turso/core/LimboDB.java

```
@@ -64,6 +65,15 @@ public static Architecture detect() {
       String osName = System.getProperty("os.name").toLowerCase();
       String osArch = System.getProperty("os.arch").toLowerCase();
 
+      // TODO: add support for arm64 on Linux
+      if (osName.contains("linux")) {
+        if (osArch.contains("aarch64") || osArch.contains("arm64")) {
+          return LINUX_X86;
```

You need to throw an exception here?

---

Yep, totally better